test/e2e/Progress.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+/* eslint-disable
+  no-undef
+*/
+const fs = require('fs');
+const { resolve } = require('path');
+const testServer = require('../helpers/test-server');
+const reloadConfig = require('../fixtures/reload-config/webpack.config');
+const runBrowser = require('../helpers/run-browser');
+
+const cssFilePath = resolve(__dirname, '../fixtures/reload-config/main.css');
+
+describe('client progress', () => {
+  describe('using hot', () => {
+    beforeAll((done) => {
+      fs.writeFileSync(
+        cssFilePath,
+        'body { background-color: rgb(0, 0, 255); }'
+      );
+      const options = {
+        port: 9000,
+        host: '0.0.0.0',
+        inline: true,
+        hot: true,
+        progress: true,
+        watchOptions: {
+          poll: 500,
+        },
+      };
+      testServer.startAwaitingCompilation(reloadConfig, options, done);
+    });
+
+    afterAll((done) => {
+      testServer.close(done);
+    });
+
+    describe('on browser client', () => {
+      jest.setTimeout(30000);
```

Comment:
need to delete after https://github.com/webpack/webpack-dev-server/pull/1965 is merged